src/lib/components/LighthouseScoresAudits/_styles.scss

```diff
@@ -0,0 +1,166 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/dimensions';
+@import '../../../styles/settings/type';
+@import '../../../styles/tools/mixins';
+
+// =============================================================================
+// LIGHTHOUSE SCORES OVERVIEW
+//
+// Shows a list of audits and what guides you could follow to fix them.
+//
+// =============================================================================
+
+web-lighthouse-scores-audits {
+  display: block;
+  min-height: 400px; // Give layout before CE upgrade.
+  margin-top: 40px;
+
+  $impactChipWidth: 75px;
+  $rowPadding: 16px;
+  $FILTERFONT: (
+    FONT_WEIGHT: 500,
+    FONT_SIZE:   14,
+    LINE_HEIGHT: 16,
+    FONT_FAMILY: $PRIMARY_FONT,
+  );
+
+  .lh-audit-list-headers {
+    @include font($FILTERFONT);
+    color: $GREY_700;
+    text-transform: uppercase;
+    display: flex;
+    padding: $rowPadding / 2 $rowPadding;
+
+    .lh-audit-list-row__item {
+      // Enforce vertical alignment w/o flex.
+      line-height: 60px;
+    }
+  }
+
+  .lh-audit-list-rows {
+    .lh-audit-list-row {
+      background-color: $WHITE;
+      margin: 8px 0;
+      padding: $rowPadding;
+      border-radius: 2px;
+      @include elevation(1);
+      display: flex;
+      align-items: center;
+      font-size: 14px;
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+
+    .lh-audit-list-row__impact {
+      display: inline-flex;
+      background-color: transparentize($WEB_EXTENDED_TERTIARY_COLOR, .8);
+      color: $WEB_EXTENDED_TERTIARY_COLOR;
+      font-weight: 600;
```

Comment:
Done. I've found this bite me a bit in personal sites, I think some browsers (Firefox) try to interpolate missing weights, but Chrome just clamps to the nearest it can find.